.pre-commit-config.yaml

```diff
@@ -8,28 +8,26 @@ ci:
     skip: []
     submodules: false
 repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.4.2
+  hooks:
+    # Run the linter.
+    - id: ruff
+      args: [--line-length=80]
```

Comment:
Should we add the fix flag back in, so that easy formatting fixes are automatically made and comitted on PRs?

Reply:
yeah, sure! we can always turn it off later if it's become troublesome.